src/utils.js

```diff
@@ -428,11 +428,15 @@ export const getStatistics = (icoConfig, allLogs, stats) => {
     stats.investors.sortedByTicket
   );
 
+  console.log('get gini');
   if (stats.money.tokenIssued > 0) {
-    const tokens = Object.keys(stats.investors.senders)
-      .map(investor => stats.investors.senders[investor].tokens);
-    stats.general.giniIndex = gini.unordered(tokens);
+    // const tokens = Object.keys(stats.investors.senders)
+    //  .map(investor => stats.investors.senders[investor].tokens);
+    const tokens = sortedSenders[0].map(investor => investor.value);
+    // todo: should just rewrite gini.ordered to accept reverse ordered array
+    stats.general.giniIndex = gini.ordered(tokens.reverse());
   }
 
+  console.log('dictionaries completed');
```

Comment:
I'm not sure about leaving `console.log` in code. General approach is that is should not be here and eslint mark those as warnings.